tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -298,3 +300,27 @@ def test_pv_provisioning_under_degraded_state_stop_rook_operator_pod_node(
 
         # Checking cluster and Ceph health
         self.sanity_helpers.health_check()
+
+    @bugzilla('1873938')
+    @skipif_no_lso
+    def test_pv_after_reboot_node(self, nodes):
+        """
+        Verify unexpected PV is not created after node reboot on LSO cluster
+
+        """
+        pv_before_reset = get_pv_names()
+        worker_node = get_typed_nodes(
+            node_type=constants.WORKER_MACHINE, num_of_nodes=1
```

Comment:
We are checking this only for one node restart, can we do rolling reboot of all worker nodes so that the disks moves around more between the nodes? 
WDYT?

Reply:
@PrasadDesala 
This test will take a long time...

Reply:
yea :) this issue is seen after multiple node reboots, so better we cover the same.